Update administrative-templates-import-custom.md (#3356)
MicrosoftDocs/memdocs: memdocs/intune/configuration/administrative-templates-import-custom.md

- Currently, the combo box setting type isn't supported. ADMX files with the combo box setting type will fail to import. All other setting types are supported.

> - Not all areas of the registry can be set using custom ADMX. For more information go to: https://learn.microsoft.com/en-us/windows/client-management/win32-and-centennial-app-policy-configuration#overview

Reviewer: ```suggestion
- Not all areas of the registry can be set using custom ADMX. For more information, see [Win32 and Desktop Bridge app ADMX policy Ingestion Overview](/windows/client-management/win32-and-centennial-app-policy-configuration#overview).
```

> - ADMX settings that are built into Windows (located at in the C:\Windows\PolicyDefinitions folder) are enabled through configuration service providers (CSP). These should not be imported using the import ADMX feature. The settings catalog or a custom policy should be used instead of ADMX import. More information on these settings here:
> - ADMX-backed policies in Policy CSP
> - https://learn.microsoft.com/en-us/windows/client-management/mdm/policies-in-policy-csp-admx-backed

Reviewer: ```suggestion
- ADMX settings that are built into Windows (located in the `C:\Windows\PolicyDefinitions` folder) are enabled through configuration service providers (CSPs). These should not be imported using the import ADMX feature. The settings catalog or a custom policy should be used instead of ADMX import. For more information on these settings, see [ADMX-backed policies in Policy CSP](/windows/client-management/mdm/policies-in-policy-csp-admx-backed).
```
